### DISCUSSION TITLE
Share Tips for Reducing GIF File Size

### DISCUSSION THREAD

**Discussion by Welding-Torch** [NickeManarin/ScreenToGif#1281] [Q&A]
Hi there, I've been using ScreenToGif for years now. But the one persistent problem I have with it is this: The GIF File sizes are too large. Even a GIF of an 8 second long video comes to 16 MB, which is too much IMHO. 

So, I think it'd be great if users of the app could share their tips and tricks for reducing GIF file sizes.

**Comment by q2333gh** [comment]
Yes ! the same tips inneed~ Most common memes gif is small and clear enough. 
We want to make that our own using Screen2Gif

**Comment by Welding-Torch** [comment]
I find that using `ffmpeg` for processing the GIF reduces file size compared to using the `ScreenToGif` processor in the settings of the app.

**Reply by q2333gh** [reply]
But the ScreenToGif   offcial site of here https://www.screentogif.com/how-to-use 
It output is so compact . maybe its because gif content too simple ? 
And its` ui is also different from latest version

**Reply by Welding-Torch** [reply]
ffmpeg is just better, and makes GIFs of lower file size.

**Reply by q2333gh** [reply]
ok ill try it with help of gpt write compress cmd hah.
